rest/permissions.py

```diff
@@ -105,7 +106,7 @@ def has_permission(self, request, view):
         except PermissionDenied:
             return False
 
-        if request.method in SAFE_METHODS:
+        if request.method in self.SAFE_METHODS_CUSTOM:
```

Comment:
I am not sure what you mean: the class `ManageTaxonomyPermission` that you modified here implements exactly the check you need (for the rest APIs).

The permission itself already exists and in this case it is checked later in the `has_permission` method of this class: see line [119](https://github.com/mitodl/lore/blob/feature/aq/edit_vocabulary/rest/permissions.py#L119) of your file